templates/zerver/documentation_main.html

```diff
@@ -11,6 +11,7 @@
     <div class="sidebar">
         <div class="content">
             <h1><a href="https://zulip.com" class="no-underline">Zulip homepage</a></h1>
+            <h1><a href="{{ doc_root }}" class="no-underline">{{ doc_root_title }}</a></h1>
```

Comment:
I think we want this to be called "Index", if we're going to have it, but I'm not sure it'll look good as a `h1`.  Can you post a screenshot?  I'm pretty sure we removed this for visual design reasons.

Reply:
I was just about to post one. :)
<img width="262" alt="help_sidebar" src="https://user-images.githubusercontent.com/7251823/127400183-b27c04e8-676d-41ad-bfca-b9dc908d082e.png">

Reply:
I don't mind "Index", but I thought mirroring what the link text says after the logo might be more consistent.